10-JavaScript/exercises/ruby-to-js-worksheet.md

```diff
@@ -45,13 +43,6 @@ Some of the later problems use Ruby `each` loops. For now, solve them using `for
 
 1. &nbsp;
     ```ruby
-    (1..3).each do |i|
-      puts i * i
-    end
```

Comment:
What is your reasoning for taking these problems out? Too many questions asking the same thing?

Reply:
Yes - felt like they were doing unnecessary work as almost exactly the same questions